testsuite/tests/lib-unix/realpath/test.ml

```diff
@@ -0,0 +1,25 @@
+(* TEST
+* hasunix
+include unix
+** bytecode
+** native
+*)
+
+let main () =
```

Comment:
`GetFinalPathNameByHandleW` returns filenames in expanded form prefixed `\\?\` which shouldn't be returned by the OCaml function. This would test for this (the `lowercase_ascii` is because `Sys.getcwd` is not normalised)
```suggestion
let main () =
  let cwd = String.lowercase_ascii (Sys.getcwd ()) in
  assert (cwd = String.lowercase_ascii (Unix.realpath cwd));
```

Reply:
Done in ea07e45